atlp-rwanda/elites-bn-be
# ft-#181339634-chat feature

# What does this PR do?
This PR if for chat feature ,where Users should be able to chat on Barefoot Nomad
# Description of Task to be completed?
-online user's can post public chat. 
# How should this be manually tested?
- clone the repository
- run yarn dev
- navigate this endpoint [link ](http://localhost:3000/api/v1/chat)
- open double screen on your browser, open the same tab with the link above, in that screens
- check the chat messages send from both users
## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration


# Any background context you want to provide?

# What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/n/projects/2555282/stories/181339634
# Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2022-04-03 at 10 45 20" src="https://user-images.githubusercontent.com/79087014/161419642-6a898084-0548-4c9a-8f22-92e238c715d2.png">

# Questions:
i need your ideas and feedback
